src/NServiceBus.Core/Encryption/ConfigureRijndaelEncryptionService.cs

```diff
@@ -12,6 +13,8 @@ namespace NServiceBus
     /// </summary>
     public static partial class ConfigureRijndaelEncryptionService
     {
+        static readonly ILog Log = LogManager.GetLogger("NServiceBus.Settings.ConfigureRijndaelEncryptionService");
```

Comment:
why do we log this with a different namespace instead of using GetLogger<Configure...>()?